pkg/controllers/state/suite_test.go

```diff
@@ -1517,9 +1517,9 @@ var _ = Describe("Node Resource Level", func() {
 
 		// Expect that the node is now nominated
 		Expect(ExpectStateNodeExists(node).Nominated()).To(BeTrue())
-		time.Sleep(time.Second * 5) // nomination window is 10s so it should still be nominated
+		time.Sleep(time.Second * 10) // nomination window is 10s so it should still be nominated
```

Comment:
Nomination window is twice the max batch I believe. https://github.com/aws/karpenter-core/blob/a4c05cede32f6210fb3df7437f9468258936c8a8/pkg/controllers/state/statenode.go#L391

I think code is right but the comment is misleading.